app/retail/templates/ambassadors.html

```diff
@@ -0,0 +1,107 @@
+{% comment %}
+  Copyright (C) 2018 Gitcoin Core
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published
+  by the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program. If not, see <http://www.gnu.org/licenses/>.
+{% endcomment %}
+{% load i18n static %}
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name="title" content="Ambassador Program | Gitcoin">
+    <meta name="description" content="Community is our priority. Join our Ambassador Program and help grow open source!">
+    {% include 'shared/head.html' %}
+    {% include 'shared/cards.html' %}
+  </head>
+  <body class="interior {{ active }} g-font-muli">
+    <div class="container-fluid header dash">
+      {% include 'shared/nav.html' %}
+    </div>
+    <div class="container">
+      <div class="row">
+        <img class="animated fadeIn src="{% static "v2/images/ambassadors.png"%}">
+      </div>
+      <div class="row">
+        <div class="col">
+          <h1>Become a Gitcoin Ambassador</h1>

```

Comment:
Can you wrap the copy throughout this file in `{% trans "copy" %}` - for example here:  `{% trans "Become a Gitcoin Ambassador" %}`